### PR TITLE
POL-570 straight line forecast

### PR DESCRIPTION
**Pull request by rshade**
### Description
This policy produces a straight-line forecast. 

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/forecasting/straight_line_forecast/straight_line_forecast.pt [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+    var end_date = getFormattedDailyDate(getEndDate(new Date(), month_counter));
+    month_list.push({
+      'start_date': start_date,
+      'end_date': end_date,
+    });
+  });
+EOS
+end
+
+script "new_costs_request", type: "javascript" do
```
Comment (nia-vf1) [inline cost/forecasting/straight_line_forecast/straight_line_forecast.pt:187]: missing "js_" prefix

File: cost/forecasting/straight_line_forecast/straight_line_forecast.pt [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+end
+
+datasource "ds_past_month_list" do
+  run_script $js_generate_past_month_list, $param_lookback_months
+end
+
+datasource "ds_costs" do
+  iterate $ds_past_month_list
+  request do
+    run_script $new_costs_request, rs_org_id, $ds_billing_centers, $param_cost_metric, $param_billing_centers, val(iter_item, 'start_date'), val(iter_item, 'end_date')
```
Comment (nia-vf1) [inline cost/forecasting/straight_line_forecast/straight_line_forecast.pt:125]: add "js_" prefix to $new_costs_request as per comment on Line 187

File: cost/forecasting/straight_line_forecast/straight_line_forecast.pt [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+  var forecasted_months = ds_make_data_package[0].forecasted_months;
+  var end_cost = costs[0].cost;
+  var start_cost = costs[costs.length - 1].cost;
+  var percent_difference = ((end_cost / start_cost) - 1);
+  var monthly_percent_difference = (percent_difference / costs.length);
+  var forecasted_cost = costs[0].cost;
+  var chartDataArray = [];
+  var XAxisArray = [];
+  var monthNames = ['January', 'February', 'March', 'April', 'May', 'June',
+    'July', 'August', 'September', 'October', 'November', 'December',
```
Comment (nia-vf1) [inline cost/forecasting/straight_line_forecast/straight_line_forecast.pt:306]: Got a comma at the end of the array after 'December', probably not a code breaker though.

Comment (rshade) [reply cost/forecasting/straight_line_forecast/straight_line_forecast.pt:306]: eslint made me :)

Comment (rshade) [reply cost/forecasting/straight_line_forecast/straight_line_forecast.pt:306]: https://eslint.org/docs/rules/comma-dangle

**Review by nia-vf1 — APPROVED** [review]
LGTM

**Review by nia-vf1 — APPROVED** [review]

**Pull request merged**